### PR TITLE
Finish homework one

### PR DESCRIPTION
**Pull request by pipeandpilcrow**
Describe what problem they’re solving, how they’re thinking about it, and why it’s a good solution:

### CODE REVIEW

**Comment by nikosolihin** [comment]
Here's my comment on this PR

File: README.md [hunk 1]
```diff
@@ -2,4 +2,6 @@
 
 This is what I added as part of my homework assignment.
 
-Another change to my homework.
\ No newline at end of file
+Another change to my homework.
+
```
Comment (nikosolihin) [inline README.md]:
```suggestion
I suggest deleting this empty line
```

File: README.md [hunk 2]
```diff
@@ -2,4 +2,6 @@
 
 This is what I added as part of my homework assignment.
```
Comment (nikosolihin) [inline README.md]: Hmm

File: README.md (hunk 1 above)
Comment (nikosolihin) [inline README.md]: ;Lorem ipsum dolor sit amet, consectetur adipisicing elit. Rem pariatur provident eaque, obcaecati laudantium officia dolores maxime illum assumenda. Dolor tenetur saepe voluptatem iure suscipit hic labore, consequuntur cum at.

File: README.md [hunk 3]
```diff
@@ -2,4 +2,6 @@
```
Comment (nikosolihin) [inline README.md]: What's this?

File: README.md [hunk 4]
```diff
@@ -2,4 +2,6 @@
 
 This is what I added as part of my homework assignment.
```
Comment (nikosolihin) [inline README.md]: This is a review comment for line 3

File: README.md (hunk 3 above)
Comment (pipeandpilcrow) [reply README.md]: Umm, I dunno

**Pull request merged**